app/_data/docs_nav_gateway_3.0.x.yml

```diff
@@ -114,19 +114,19 @@ items:
       - text: Running Kong
         items:
           - text: Running Kong as a non-root user
-            url: /production-deployment/running-kong/kong-user
+            url: /production/running-kong/kong-user
           - text: Securing the Admin API
-            url: /production-deployment/running-kong/secure-admin-api
+            url: /production/running-kong/secure-admin-api
           - text: Using systemd
-            url: /production-deployment/running-kong/systemd
+            url: /production/running-kong/systemd
       - text: Access Control
         items:
           - text: Start Kong Gateway Securely
-            url: /production-deployment/access-control/start-securely
+            url: /production/access-control/start-securely
           - text: Programatically Creating Admins
-            url: /production-deployment/access-control/register-admin-api
+            url: /production/access-control/register-admin-api
           - text: Enabling RBAC
-            url: /production-deployment/access-control/enable-rbac
+            url: /production/access-control/enable-rbac
       - text: Licenses
```

Comment:
All of the pages in this section are missing "/production/" in the URL and "Production Deployment" in the breadcrumbs. I'm not sure if this is intentional or not (I'm guessing it's unintentional because these pages were moved here).

Reply:
This is intentional. Licenses were moved to the top level as licenses are technically used in dev as well as production, but kept in the Kong in Production section in the sidenav.

Expected URL is `/gateway/latest/licenses`